mutiny-core/src/utils.rs

```diff
@@ -101,7 +106,7 @@ where
 {
     #[cfg(not(target_arch = "wasm32"))]
     {
-        tokio::task::spawn(future);
+        tokio::task::spawn_local(future);
```

Comment:
this didn't work for me when trying to create a mutiny-server

switching to this did 

`tokio::task::LocalSet::new().spawn_local(future);`